tools/devctr/Dockerfile

```diff
@@ -107,8 +107,10 @@ RUN cd /tmp/poetry && \
 # - Install the Rust toolchain. Kani only work on x86, so only try to install it there
 # - Build and install crosvm (used as vhost-user-blk backend)
 # - Clean up cargo compilation directories
+# - Always install both x86_64 and aarch64 musl targets, as our rust-toolchain.toml would force on-the-fly installation of both anyway
```

Comment:
Just occurred to me.. could we move devtool to always specify --privileged? Would we lose anything?

Reply:
Mh, probably not a problem, but I kinda already got annoyed by it always downloading the target when I run any docker command, so I'd like to just have both in the container anyway